test/round.test.ts

```diff
@@ -248,9 +248,42 @@ describe('RoundState', () => {
         );
       });
 
-      it.todo("doesn't override blockade");
+      it.skip("doesn't override blockade", () => {
```

Comment:
this test is also incorrect (blockade only comes into effect once there are 3 cards in the theater)

Reply:
fix'd